packages/web/docs/src/components/pricing/plans-table.tsx

```diff
@@ -189,25 +207,118 @@ export function PlansTable({ className }: { className?: string }) {
             </tr>
 
             <TableSubheaderRow
-              icon={<AvailabilityIcon />}
-              title="Availability"
-              description="Engineered for uninterrupted performance and reliability."
+              icon={<UsageIcon />}
+              title="Analytics, Monitoring & Metrics"
+              description="Monitor and evolve your schema in a flexible way."
             />
+
             <tr>
-              <PlansTableCell>99.95% uptime of operation collection</PlansTableCell>
+              <PlansTableCell className="whitespace-pre">Operations per month</PlansTableCell>
               <PlansTableCell activePlan={activePlan} plan="Hobby">
-                {NO}
+                1M operations per month
               </PlansTableCell>
               <PlansTableCell activePlan={activePlan} plan="Pro">
-                {NO}
+                1M operations per month
+                <br className="max-sm:inline" />
+                <span className="font-normal">Then $10 per 1 million operations</span>
+              </PlansTableCell>
+              <PlansTableCell activePlan={activePlan} plan="Enterprise">
+                Custom
```

Comment:
Should we mention what we say in meetings?
That we use Enterprise to give you a discount, not as a tax?